distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -73,21 +73,24 @@ org.codice.ddf.system.registry-id=
 # Solr Client Configuration
 #
 
+# Disable Solr Config API to limit potential vulnerabilities
+disable.configEdit=true
```

Comment:
Not a very good namespace is it. Good thing it has a note with it :)

Reply:
Yeah, I had the same thought :-)